src/Whoops/Handler/JsonResponseHandler.php

```diff
@@ -21,11 +21,6 @@ class JsonResponseHandler extends Handler
     private $returnFrames = false;
 
     /**
-     * @var bool
```

Comment:
@denis-sokolov I assume a major version bump, therefore broke BC to cleanup the handler. In case you wanne preserve BC give me a ping, so I will re-add and deprecate this stuff instead.